packages/botonic-core/src/hubtype-service.js

```diff
@@ -177,6 +181,20 @@ export class HubtypeService {
     )
   }
 
+  destroyPusher() {
+    if (!this.pusher) return
```

Comment:
shouldn't we call destroyPusher from cleanAndReject in line 75?

Reply:
calling it now